components/style/values/specified/position.rs

```diff
@@ -145,12 +271,14 @@ impl PositionComponent {
     #[inline]
     pub fn to_length_or_percentage(self) -> LengthOrPercentage {
         match self {
-            PositionComponent::LengthOrPercentage(value) => value,
-            PositionComponent::Center => LengthOrPercentage::Percentage(Percentage(0.5)),
-            PositionComponent::Left |
-            PositionComponent::Top => LengthOrPercentage::Percentage(Percentage(0.0)),
-            PositionComponent::Right |
-            PositionComponent::Bottom => LengthOrPercentage::Percentage(Percentage(1.0)),
+            PositionComponent::Length(value) => value,
+            PositionComponent::Keyword(keyword) if keyword == Keyword::Center =>
+                LengthOrPercentage::Percentage(Percentage(0.5)),
+            PositionComponent::Keyword(keyword) if keyword == Keyword::Left ||
+            keyword == Keyword::Top => LengthOrPercentage::Percentage(Percentage(0.0)),
+            PositionComponent::Keyword(keyword) if keyword == Keyword::Right ||
+            keyword == Keyword::Bottom => LengthOrPercentage::Percentage(Percentage(1.0)),
+            PositionComponent::Keyword(_) => unimplemented!(), // TODO: All keywords are covered but rust forcing me to add this too?
```

Comment:
Maybe `match` further on `keyword` looks better and eliminate the compiler's complain?